src/api/test/functional/webui/search_controller_test.rb

```diff
@@ -112,13 +112,22 @@ def test_header_search_functionality
     page.must_have_text(/Base.* distro without update project/)
   end
 
+  def test_search_by_request_number
+    visit root_path
+    fill_in 'search', with: '#1'
+    page.evaluate_script("$('#global-search-form').get(0).submit()")
+
+    page.must_have_text(/Request 1/)
+  end
+
   def test_search_by_baseurl
     visit root_path
-    fill_in 'search', with: 'obs://build.opensuse.org/openSUSE:Factory/standard/fd6e76cd402226c76e65438a5e3df693-bash'
+    fill_in 'search', with: 'obs://myhost/BaseDistro/BaseDistro_repo/d430c2f61e4d8999f9ca6ed6667a104e-pack2'
```

Comment:
Line is too long. [108/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.